### PR TITLE
Run automated test suite with GitHub Actions

### PR DESCRIPTION
**Pull request by jpgill86**

### CODE REVIEW

File: ephyviewer/tests/testing_tools.py [hunk 1]
```diff
@@ -148,3 +148,23 @@ def make_fake_spiketrain_source():
...
+        'aep_05/Block-1/aep_05_Block-1.tsq',
+
+        # second Neo Segment -- not yet used by ephyviewer
+        # 'aep_05/Block-2/aep_05_Block-2.Tbk',
+        # 'aep_05/Block-2/aep_05_Block-2.Tdx',
+        # 'aep_05/Block-2/aep_05_Block-2.tev',
+        # 'aep_05/Block-2/aep_05_Block-2.tsq',
+    ]
+    t.setUp()
+    return t.local_test_dir
```
Comment (jpgill86) [inline ephyviewer/tests/testing_tools.py:170]: These data files downloaded from [GIN](https://gin.g-node.org/NeuralEnsemble/ephy_testing_data) for testing are kind of big (~65 MB). Tests might be faster if a smaller data set was used instead, but for now it's not too bad.

Comment (samuelgarcia) [reply ephyviewer/tests/testing_tools.py:170]: Is there any cache possibility when downloading something in GH aactiosn ?

Comment (jpgill86) [reply ephyviewer/tests/testing_tools.py:170]: Great idea! I've implemented it in #145.

**Comment by samuelgarcia** [comment]
Tu es incroyable!!
Merci.

**Pull request merged**